src/js/hca-rjsf/config/form.js

```diff
@@ -69,9 +73,94 @@ const formConfig = {
       title: 'Veteran Information',
       pages: {
         veteranInformation: {
-          path: 'veteran/information',
+          path: 'veteran-information/personal-information',
           title: 'Veteran information',
           initialData: {},
+          uiSchema: {
+            veteranFullName: _.merge(fullNameUISchema, {
+              last: {
+                'ui:errorMessages': {
+                  minLength: 'Please provide a valid name. Must be at least 2 characters.'
+                }
+              }
+            }),
+            mothersMaidenName: {
+              'ui:title': 'Mother’s maiden name'
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              veteranFullName: _.merge(fullName, {
+                properties: {
+                  suffix: {
+                    type: 'string'
+                  }
+                }
+              }),
+              mothersMaidenName: {
+                type: 'string'
+              }
+            }
+          }
+        },
+        birthInformation: {
+          path: 'veteran-information/birth-information',
+          title: 'Veteran information',
+          initialData: {},
+          uiSchema: {
+            veteranDateOfBirth: {
+              'ui:title': 'Date of birth'
+            },
+            veteranSocialSecurityNumber: ssn.uiSchema,
+            'view:placeOfBirth': {
+              'ui:title': 'Place of birth',
+              cityOfBirth: {
+                'ui:title': 'City'
+              },
+              stateOfBirth: {
+                'ui:title': 'State'
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            required: ['veteranDateOfBirth', 'veteranSocialSecurityNumber'],
+            properties: {
+              veteranDateOfBirth: date,
+              veteranSocialSecurityNumber: ssn.schema,
+              'view:placeOfBirth': {
+                type: 'object',
+                properties: {
+                  cityOfBirth,
+                  stateOfBirth: {
+                    type: 'string',
+                    'enum': states.USA.map(state => state.value),
+                    enumNames: states.USA.map(state => state.label)
```

Comment:
It doesn't save you that much, I guess, but this file has a `stateLabels` object that you could use in `ui:options.labels`. See the `vaFacility` config for an example. 

That would remove the need for `enumNames` and save the extra iteration of the states list.